src/Update.cpp

```diff
@@ -1353,8 +1353,8 @@ static void SusceptibleToRecovered(int cellIndex)
 
 	// assert values are non-negative
 	assert(Cells[cellIndex].S >= 0);
-	assert(Cells[cellIndex].latent >= 0);
-	assert(Cells[cellIndex].infected >= 0);
+	assert(Cells[cellIndex].latent >= static_cast<int*>(0));
+	assert(Cells[cellIndex].infected >= static_cast<int*>(0));
```

Comment:
Should we remove the asserts against int* pointers? They were only added as an extra while asserting the severity states, which are all integers.